conda_smithy/lint_recipe.py

```diff
@@ -149,6 +151,15 @@ def lintify(meta, recipe_dir=None):
     except RuntimeError as e:
         lints.append(str(e))
 
+    # 13: Subheaders should be in the allowed subheadings
```

Comment:
Sure, do you know why the existing test are failing? https://travis-ci.org/conda-forge/conda-smithy/jobs/247245661#L438